src/databricks/labs/ucx/hive_metastore/tables.scala

```diff
@@ -78,8 +79,16 @@ def getInventoryDatabase(): String={
 }
 
 val inventoryDatabase = getInventoryDatabase()
-val df = metadataForAllTables(spark.sharedState.externalCatalog.listDatabases(), failures)
+var df = metadataForAllTables(spark.sharedState.externalCatalog.listDatabases(), failures)
+var columnsToMap = df.columns
+columnsToMap.map(column => {
```

Comment:
Why do we need this change?

Reply:
this is required because in /queries/views/objects.sql there is join between tables and failures on catalog, database, name. I have updated the script to only lower case these 3 fields for the table_failures table